apps/precinct-scanner/src/screens/UnconfiguredElectionScreen.tsx

```diff
@@ -190,13 +195,3 @@ const UnconfiguredElectionScreen = ({
 }
 
 export default UnconfiguredElectionScreen
-
-/* istanbul ignore next */
-export const DefaultPreview = (): JSX.Element => {
```

Comment:
I don't understand what this is, prettier automatically deleted it for me for some reason?

Reply:
Put it back? Or does it keep deleting it?

Reply:
Well it doesn't seem to be used anywhere so I wasn't sure what the point of it was? But I can add back.

Reply:
It's used by `PreviewApp`.